src/views/Pools/components/PoolCard/Modals/CollectModal.tsx

```diff
@@ -107,7 +108,7 @@ const CollectModal: React.FC<CollectModalProps> = ({
           <Flex ml="10px" ref={targetRef}>
             <HelpIcon color="textSubtle" />
           </Flex>
-          {tooltipVisible && tooltip}
+          {tooltipVisible && ReactDOM.createPortal(tooltip, document.getElementById('root'))}
```

Comment:
This is a good example that probably not the best idea to put everything in Portal by default 😄 
Can be left as it was
![Screenshot 2021-05-21 at 13 55 22](https://user-images.githubusercontent.com/81824236/119126861-45654180-ba3c-11eb-9b2c-7b848b91ec5c.png)

Reply:
done